src/Admin/AbstractAdmin.php

```diff
@@ -759,6 +759,12 @@ public function getFilterParameters()
         // build the values array
         if ($this->hasRequest()) {
             $filters = $this->request->query->get('filter', []);
+            if (!empty($filters['_page'])) {
```

Comment:
should this really be checked with `empty`? Also, can you please update the tests to cover this?

Reply:
yes, otherwise `Notice: Undefined index: _per_page`

Reply:
`isset` `array_key_exists` are also options, empty looks strange to me as I never used it for checking the key

Reply:
It may happen that the `_per_page` will be empty.


```
if (isset($filters['_per_page'])) {
    $filters['_per_page'] = (int) $filters['_per_page'];
}
```

before
`"_per_page" => ""`

after
`"_per_page" => 0`


is it OK?

Reply:
I changed it.